### PR TITLE
Add code coverage for JavaScript tests #5040

### LINKED ISSUE #5040: Add code coverage for JavaScript tests
Since we are already using QUnit to run JavaScript tests, it's good to also provide information for code coverage, so we would know where can be improved in our code, or add missing test cases if needed.
[Blanket.js](http://blanketjs.org/) seems like a good choice as it is lightweight, works well, and supports QUnit by default. It's also very easy to configure to the current system as well.

### PR DESCRIPTION
**Pull request by Gisonrg**
Fixes #5040 
Use [Blanket.js](https://github.com/alex-seville/blanket/blob/master/docs/getting_started_browser.md). It's easier to configure on the front-end comparing with other frameworks

Now one can enable/disable displaying coverage when view the result of `AllJsTest`
screen shot 2016-03-19 at 15 41 21

The coverage will look like this
screen shot 2016-03-19 at 15 41 25
screen shot 2016-03-19 at 15 52 26

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Easy to use and has high value-add, this is the kind of extension that I like :+1:

**Comment by damithc** [comment]
Perhaps we should update the 'tool stack' section in our dev docs about this new library?

**Comment by Gisonrg** [comment]
Thanks for the review :D 
Yea probably should mention the lib in the 'tool stack' section, I will add it.

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -277,6 +277,8 @@ Troubleshooting instructions are given [in this document](https://docs.google.co
     TestNG is a Java test automation framework.
 * **QUnit** [version 1.10.0]
     QUnit is a JavaScript unit test suite.
+* **Blanket.js** [version 1.2.1]
+    Blanket.js is an easy to use JavaScript code coverage library that works both in-browser and with nodejs.
```
Comment (damithc) [inline devdocs/settingUp.md]: `Blanket.js is a JavaScript code coverage library.` should be enough.

Comment (Gisonrg) [inline devdocs/settingUp.md]: Done :ok_hand:

**Pull request merged**